src/wp-includes/script-loader.php

```diff
@@ -319,6 +319,12 @@ function wp_default_packages_scripts( $scripts ) {
 			$scripts->add_inline_script( $handle, $script, 'after' );
 		}
 	}
+
+	/*
+	 * wp-nux has been deprecated then removed, an empty script is left here
+	 * to prevent breaking plugins that define it as a depdendency.
+	 */
+	$scripts->add( 'wp-nux', false );
```

Comment:
Hmm, thinking that registering a non-existing script will actually break all scripts that list it as a dependency. Also that will most likely result in a JS "fatal error" preventing all other JS from running.

The better option here would be to not register non-existing scripts. That way any other scripts that depend on this one will not be loaded which would prevent eventual "breaking" of the whole page.